lib/node-arguments.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const minimist = require('minimist');
+const omit = require('lodash/omit');
+
+/**
+ * @param {string[]} confParams
+ * @param {string} cliParams
+ * @return {object}
+ */
+function normalizeNodeArguments(confParams, cliParams) {
```

Comment:
If we can assume the arguments may be `undefined`, we can use default parameter values:

```suggestion
function normalizeNodeArguments(confParams = [], cliParams = '') {
```

And we can get rid of the JSDoc comment too.

We do need to enforce `confParams` is an array.